converter/datamodel.py

```diff
@@ -105,6 +105,24 @@ def from_json(cls, sample):
 
         return cls(alias, accession, taxon, taxonId, attributes, material_type, description)
 
+    @classmethod
+    def from_dict(cls, sample_dict):
```

Comment:
what does cls stands for everywhere? Could we have a more explicit name?

Reply:
On all the from_dict methods you're passing an object (of class `cls`?) which is named `cls`, but you don't seem to be using it, as you are always creating a new object based on the sample_dict... this pattern is a bit confusing. Could you clarify what you intend by that in some inline docs?